dts/bindings/sensor/ltr,329.yaml

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2023 Konrad Sikora
+# SPDX-License-Identifier: Apache-2.0
+
+description: Lite-on LTR329 Digital Ambient Light Sensor
+
+compatible: "ltr,329"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+  gain:
+    type: int
+    default: 0
+    description: |
+      ADC gain factor. There are 6 possible values,
+      which correspond to different gain factors:
+      - 0 # Gain 1X (1 lux to 64k lux)
+      - 1 # Gain 2X (0.5 lux to 32k lux)
+      - 2 # Gain 4X (0.25 lux to 16k lux)
+      - 3 # Gain 8X (0.125 lux to 8k lux)
+      - 6 # Gain 48X (0.02 lux to 1.3k lux)
+      - 7 # Gain 96X (0.01 lux to 600 lux)
+    enum: [0, 1, 2, 3, 6, 7]
+
+  integration-time:
+    type: int
+    default: 0
```

Comment:
Need to justify in the description why this default value was chosen. Is it a register reset value?

Reply:
register reset value, comment added